mmv1/products/privateca/api.yaml

```diff
@@ -165,12 +165,22 @@ objects:
                 description: |
                   Describes values that are relevant in a CA certificate.
                 properties:
+                  - !ruby/object:Api::Type::Boolean
+                    name: 'nonCa'
+                    url_param_only: true
+                    description: |
+                      Must be set to true when is_ca is set to false.
```

Comment:
These descriptions are incorrect now. It would be helpful to include information about why a user would want to use these fields, and what it actually does on the resource it creates.

Reply:
Hey @slevenick , descriptions are updated to improve the usability.